packs/spell-effects/spell-effect-mantle-of-the-melting-heart.json

```
@@ -0,0 +1,228 @@
+                            "melting-heart-second:copper-core"
+                        ]
+                    }
+                ],
+                "type": "electricity",
+                "value": 10
+            },
+            {
+                "key": "Immunity",
+                "predicate": [
+                    {
+                        "or": [
+                            "melting-heart-first:golden-flesh",
+                            "melting-heart-second:golden-flesh"
+                        ]
+                    }
+                ],
+                "type": [
+                    "disease",
+                    "poison"
+                ]
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "against-poison-disease-onself",
+                    {
+                        "or": [
+                            "melting-heart-first:golden-flesh",
+                            "melting-heart-second:golden-flesh"
+                        ]
+                    }
+                ],
+                "selector": [
+                    "saving-throw",
+                    "skill-check"
+                ],
+                "value": 2
+            },
+            {
+                "category": "unarmed",
+                "damage": {
+                    "base": {
+                        "damageType": "bludgeoning",
+                        "dice": 1,
+                        "die": "d4"
+                    }
+                },
+                "group": "flail",
+                "key": "Strike",
+                "label": "PF2E.SpecificRule.SpellEffectMantle.MeltingHeart.CabledFist.Label",
+                "predicate": [
+                    {
+                        "or": [
+                            "melting-heart-first:weighted-grasp",
+                            "melting-heart-second:weighted-grasp"
+                        ]
+                    }
+                ],
+                "slug": "cabled-fist",
+                "traits": [
+                    "disarm",
+                    "finesse",
+                    "reach",
+                    "trip",
+                    "unarmed"
+                ]
+            },
+            {
+                "damageType": "poison",
+                "diceNumber": 1,
+                "dieSize": "d6",
+                "key": "DamageDice",
+                "predicate": [
+                    {
+                        "or": [
+                            "melting-heart-first:weighted-grasp",
+                            "melting-heart-second:weighted-grasp"
+                        ]
+                    }
+                ],
+                "selector": "cabled-fist-damage"
+            },
+            {
+                "damageType": "acid",
+                "diceNumber": 2,
+                "dieSize": "d6",
+                "hideIfDisabled": true,
+                "key": "DamageDice",
```

```suggestion
                "key": "DamageDice",
                "label": "PF2E.SpecificRule.SpellEffectMantle.MeltingHeart.Label.ReactiveTouch",
```
Might as well label it since we have the string. Also, maybe it's a good idea to not hide it so a GM can make a judgement call on whether the damage can be added in other circumstances of metal-themed creatures without necessarily having the trait.